scripts/prove_stdio.sh

```diff
@@ -95,14 +95,21 @@ fi
 # proof. This is useful for quickly testing decoding and all of the
 # other non-proving code.
 if [[ $TEST_ONLY == "test_only" ]]; then
-    cargo run --quiet --release --package zero --bin leader -- --test-only --runtime in-memory --load-strategy on-demand --block-batch-size $BLOCK_BATCH_SIZE --proof-output-dir $PROOF_OUTPUT_DIR stdio < $INPUT_FILE &> $TEST_OUT_PATH
-    if grep -q 'All proof witnesses have been generated successfully.' $TEST_OUT_PATH; then
+    cargo run --quiet --release --package zero --bin leader -- \
+      --test-only \
+      --runtime in-memory \
+      --load-strategy on-demand \
+      --block-batch-size "$BLOCK_BATCH_SIZE" \
+      --proof-output-dir "$PROOF_OUTPUT_DIR" \
+      stdio < "$INPUT_FILE" &> "$TEST_OUT_PATH"
+
+    if grep -q 'All proof witnesses have been generated successfully.' "$TEST_OUT_PATH"; then
         echo -e "\n\nSuccess - Note this was just a test, not a proof"
-        rm $TEST_OUT_PATH
+        rm "$TEST_OUT_PATH"
         exit
     else
         # Some error occurred, display the logs and exit.
-        cat $OUT_LOG_PATH
+        cat "$OUT_LOG_PATH"
         echo "Failed to create proof witnesses. See $OUT_LOG_PATH for more details."
```

Comment:
Unrelated (also changed in #734 but this one would land before)
```suggestion
        cat "$TEST_OUT_PATH"
        echo "Failed to create proof witnesses. See $TEST_OUT_PATH for more details."
```